ffi/athcon/bindings/rust/athcon-client/Cargo.toml

```diff
@@ -8,7 +8,10 @@ license.workspace = true
 edition.workspace = true
 
 [dependencies]
+athena-vmlib = { path = "../../../../vmlib" }
```

Comment:
This makes me slightly uncomfortable. In my mental map of the codebase, the athcon bindings shouldn't rely on anything external; they're meant to be imported, and to never create a circular dependency. Does this bother you at all?

Reply:
Actually, it does, although I don't understand the part about circular dependencies. I feel the Rust bindings should probably use the same mechanics as the Go ones. I.e. they should open a shared library provided at a given path and call `athcon_create_athenavmwrapper()` from it.